ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -830,19 +831,17 @@ export class IndexerClient {
     extraArgs?: {
       options?: IndexerPaginationArgs;
       orderBy?: IndexerSortBy<Current_Fungible_Asset_Balances_Order_By>[];
+      filter?: [Current_Fungible_Asset_Balances_Bool_Exp];
```

Comment:
I think we need to default to filtering out secondary stores. Otherwise the fungible_asset_client also needs to be rewritten to allow transferring/checking balances from secondary stores. Currently no operations regarding secondary stores are supported. This means developers can run into a very weird experience here:
1. They call check getAccountCoinsData, which returns secondary stores
2. They sum up the balances and display to the users which now show balances > what they have in primary stores
3. They try to send > what they have in primary store => Errors
4. Everyone is now confused. Users are confused that they have less money than what wallets show them. Developers are confused why things error out